server/core/config/valid/global_cfg_test.go

```diff
@@ -180,6 +182,13 @@ func TestNewGlobalCfg(t *testing.T) {
 			unDivergedReq:      true,
 			policyCheckEnabled: false,
 		},
+		{
+			allowRepoCfg:       true,
+			approvedReq:        true,
+			mergeableReq:       true,
+			unDivergedReq:      true,
+			policyCheckEnabled: false,
+		},
 		{
```

Comment:
The above test case seems like a duplicate of lines 178 to 184

Reply:
Removed the duplicate test.